pyproject.toml

```diff
@@ -18,51 +16,47 @@ packages = [
     { include = "nautobot_circuit_maintenance" },
 ]
 
-[[tool.poetry.source]]
-name = "pypi-legacy"
-url = "https://pypi.org/simple/"
-secondary = true
-
 [tool.poetry.dependencies]
-python = "^3.7"
-nautobot = "^1.4.0"
-nautobot-capacity-metrics = {version = "*", optional = true }
 bs4 = "^0.0.1"
 circuit-maintenance-parser = "^2.0.0"
 google-api-python-client = "^2.9.0"
-google-oauth = { version = "^1.0.0", source = "pypi-legacy" }
 google-auth-httplib2 = "^0.1.0"
 google-auth-oauthlib = "^0.5.0"
-# Added due an issue with Markdown 3.3.5
-markdown = "!=3.3.5"
+markdown = "!=3.3.5"  # Added due an issue with Markdown 3.3.5
+nautobot = { version = "^1.4.0", optional = true }  # Used for local development
+nautobot-capacity-metrics = {version = "*", optional = true }
+python = "^3.8"
 
 [tool.poetry.dev-dependencies]
-invoke = "*"
-# Black 23.x.x configuration changes and migration files are taken into account
-black = "^22.6.0"
-django-debug-toolbar = "*"
-yamllint = "*"
+Markdown = "*"
 bandit = "*"
+black = "*"
```

Comment:
Changing `black` from a pinned dependency `^22.6.0` to an unpinned `*` dependency seems odd to me. Should we instead be bumping it to a newer pinned version?

Reply:
this was reverted to the cookiecutter standards (initial one). I could pin a specific one if required